api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -61,10 +62,14 @@
   @Mapping(target = "researchPurpose", source = "dbWorkspace")
   Workspace toApiWorkspace(DbWorkspace dbWorkspace);
 
-  @Mapping(target = "workspace", source = "dbWorkspace")
-  @Mapping(target = "accessLevel", source = "firecloudWorkspaceResponse")
-  WorkspaceResponse toApiWorkspaceResponse(
-      DbWorkspace dbWorkspace, FirecloudWorkspaceResponse firecloudWorkspaceResponse);
+  default WorkspaceResponse toApiWorkspaceResponse(
+      DbWorkspace dbWorkspace, FirecloudWorkspaceResponse firecloudWorkspaceResponse) {
+    FirecloudMapper fcMapper = Mappers.getMapper(FirecloudMapper.class);
```

Comment:
Now that I'm in a default mapper, I needed to map the access level. This is the mechanism I could come up with for achieving this - I didn't see another built-in mechanism to inject the mapper, though I'm thinking perhaps it's be possible if I just split this mapper out from this class.

Reply:
Another useful trick  is the `@AfterMapping` facility, which works well,  but is somewhat less elegant IMHO. Don't think  you necessarily need it here.

Reply:
Right, but this is a default mapper. AFAICT the `@Mapping` annotation on a default mapper does nothing as default mappers don't even result in generated code.

Reply:
I fiddled with it for 20 minutes or so but didn't get much further. If a simple default method is working then I'd stick with that until we can overhaul the WorkspaceMapper somewhat.

Reply:
Thanks for pulling it down and testing.